### PR TITLE
Move M4A Track Definitions

### PR DESCRIPTION
**Pull request by Kurausukun**
## Description
It makes a lot more sense to define them here instead of awkwardly shoving them at the bottom of m4a_1.s, which is prone to people deleting it without noticing (which is how this PR came to be).  Also made some defines to make them easier to change. Thanks to pidgey for helping.

## **Discord contact info**
Kurausukun#9923

### CODE REVIEW

**Comment by GriffinRichards** [comment]
You need to update `ld_script_modern.txt`, it doesn't include the `.bss` section for `data`

**Comment by Kurausukun** [comment]
> You need to update `ld_script_modern.txt`, it doesn't include the `.bss` section for `data`

Fixed

File: ld_script_modern.txt [hunk 1]
```diff
@@ -27,6 +27,7 @@ SECTIONS {
         /* .bss starts at 0x3000000 */
         src/*.o(.bss);
         gflib/*.o(.bss);
+		data/*.o(.bss);
         *libc.a:*.o(.bss*);
```
Comment (GriffinRichards) [inline ld_script_modern.txt:31]: Tabs vs. spaces

Comment (Kurausukun) [reply ld_script_modern.txt]: fixed

**Pull request merged**